odoo/tools/func.py

```diff
@@ -167,12 +167,18 @@ def __str__(self): return str(self._value)
     def __bytes__(self): return bytes(self._value)
     def __format__(self, format_spec): return format(self._value, format_spec)
 
-    def __lt__(self, other): return self._value < other
-    def __le__(self, other): return self._value <= other
-    def __eq__(self, other): return self._value == other
-    def __ne__(self, other): return self._value != other
-    def __gt__(self, other): return self._value > other
-    def __ge__(self, other): return self._value >= other
+    def __lt__(self, other):
+        return other > self._value
+    def __le__(self, other):
+        return other >= self._value
+    def __eq__(self, other):
+        return other == self._value
+    def __ne__(self, other):
+        return other != self._value
+    def __gt__(self, other):
+        return other < self._value
+    def __ge__(self, other):
+        return other <= self._value
```

Comment:
Could you clarify those changes? both operands are needed to evaluate the expression, why is it suddenly necessary to swap them, and why is this *strictly better in all cases*?

Also by rewrapping you're obscuring the important part of the diff and making that block inconsistent with the surrounding code. If it's just to avoid some linting flag, it's better to ignore the warning.

Reply:
Flut, I think during a rebase I merged two commits. Indeed, the commit does not explain the change. I will split them again.